Gemfile.lock

```diff
@@ -257,7 +257,7 @@ GEM
       aws-sdk-core (~> 3, >= 3.210.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.5)
-    aws-sdk-sns (1.92.0)
+    aws-sdk-sns (1.90.0)
```

Comment:
I don't think the Gemfile should be updated in this PR

Reply:
Good catch.